particula/constants.py

```diff
@@ -1,47 +1,35 @@
-""" A centralized location for important, unchanged physics parameters.
+""" a centralized location for important, unchanged physics parameters.
 
     This file contains constants that are used in multiple modules. Each
     constant has its own units and exported with them. The constants are
     mostly related to atmospheric aerosol particles in usual conditions.
-    They are flexible enough to depend on on the temperature and other
-    environmental variablaes (defined in the _environment module). In
-    the event there is an interest in overriding the values of these
-    constants:
-
-    TODO:
-        - Add a way to override the constants by parsing a user-supplied
-            configuration file.
+
 """
 
 from particula import u
 
-# Boltzmann's constant in m^2 kg s^-2 K^-1
 BOLTZMANN_CONSTANT = 1.380649e-23 * u.m**2 * u.kg / u.s**2 / u.K
 
-# Avogadro's number
 AVOGADRO_NUMBER = 6.022140857e23 / u.mol
 
 # Gas constant in J mol^-1 K^-1
 GAS_CONSTANT = BOLTZMANN_CONSTANT * AVOGADRO_NUMBER
 
-# elementary charge in C
 ELEMENTARY_CHARGE_VALUE = 1.60217662e-19 * u.C
 
 # Relative permittivity of air at room temperature
 # Previously known as the "dielectric constant"
 # Often denoted as epsilon
 RELATIVE_PERMITTIVITY_AIR = 1.0005  # unitless
 
-# permittivity of free space in F/m
+# Permittivity of free space in F/m
 # Also known as the electric constant, permittivity of free space
 # Often denoted by epsilon_0
 VACUUM_PERMITTIVITY = 8.85418782e-12 * u.F / u.m
 
-# permittivity of air
 ELECTRIC_PERMITTIVITY = RELATIVE_PERMITTIVITY_AIR * VACUUM_PERMITTIVITY
 
-# viscosity of air at 273.15 K
-# these values are used to calculate the dynamic viscosity of air
+# These values are used to calculate the dynamic viscosity of air
```

Comment:
Doesn't need to be in this PR, but it seems like these values are dependent on temperature, so we may want to move them into a separate file!

Reply:
No no, these values are constant. Look at my other PR to get a more thorough understanding of how this works in practice, #122 or see this link which I am going to cite in the updated docstring           there https://resources.wolframcloud.com/FormulaRepository/resources/Sutherlands-Formula. The *resulting* dynamic viscosity is the one that's changing; I know, I know, such a misleading name to call it "dynamic" but oh well... 

Let me take this opportunity to actually re-answer one of your earlier questions about the need to isolate these methods. Essentially, this actually goes to illustrate why it is best to have these as isolated methods. Why? Because this particular formula actually applied to any _ideal_ gas --- and we are assuming an ideal gas behavior of air here. So what happens if we want to calculate this in a different gas (unlikely) or if we want to relax the ideal gas assumption? Having these utilities be independent and isolated will make it much easier to tweak them without having to tweak the underlying code for the calculating the end products. For example, in the following PR (#122), I add a lot of options for the user to tweak this, but I haven't added the deviation from the ideal gas yet (I actually don't know how to do it because it is so niche and unlikely to change things --- but that's where science actually happens, at the edge cases). Anyway, so hopefully this starts to make more physics sense to you.

Reply:
Oh! Maybe your point is that they're linked to temperature in a way? Yes, but they're anchored at a temperature, hence `REF` in the name. So, these are considered constants imo, but we may want to change the names slightly to reflect the specificity